app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/accountsetup/SetupAutoFillViewModel.kt

```diff
@@ -35,6 +47,12 @@ class SetupAutoFillViewModel @Inject constructor(
             }
             .onEach(::sendAction)
             .launchIn(viewModelScope)
+
+        stateFlow
+            .onEach {
+                savedStateHandle[KEY_STATE] = it
```

Comment:
Why are we adding this?

Reply:
So I wanted to ask about this, in `SetupUnlockViewModel` we have `SavedStateHandle` but are never actually setting it, and basically using it as a way to pass in and initial state in some test cases. I feel like if we are going to add the dependency we should use it in the implementation.

Reply:
No, that might be my mistake. I should have left a comment. We almost never want to actually use the `savedStateHandle`. except for tests.

Normally there would be a comment like this:
```kotlin
// We load the state from the savedStateHandle for testing purposes.
initialState = savedStateHandle[KEY_STATE] ?: run {
```

Reply:
makes sense, thanks for clarifying!